.lintstagedrc.yml

```diff
@@ -0,0 +1,7 @@
+linters:
+  lib/*.{js, ts}:
+    - "npm run lint"
```

Comment:
no! Use the `tslint -t stylish --fix` lint-staged should enumerate the files.

Reply:
If run lint equals 'tslint --project tsconfig.json -t stylish --fix', isnt it the same then?

Reply:
I guess :confused:

Reply:
Fixed :)